keps/sig-node/3008-qos-class-resources/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: QoS-class resources
+kep-number: 3008
+authors:
+  - "@marquiz"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: 2021-10-07
+reviewers:
+  - "@rphillips"
+approvers:
+  - "@sig-node-leads"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: ClassResources
```

Comment:
I'm not crazy about this feature gate name.

Reply:
Me neither 😅 Without better suggestions so far, I've stuck with the term "QoS-class resources" and also the name of the feature gate has been unchanged.

Reply:
That's kinda why I like `PrioritizedResources`.

Reply:
K, I changed this to `QOSResources` as that's the name that's currently used in the README.md. In the README.md I now added the feature gate name as UNRESOLVED. With a note that all the naming needs to be in sync with this. WDYT @logicalhan ?